src/CMakeLists.txt

```diff
@@ -59,6 +55,36 @@ set(TRITON_THIRD_PARTY_INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/third-party-i
 # Add third party installation to the search path
 set(CMAKE_PREFIX_PATH ${TRITON_THIRD_PARTY_INSTALL_PREFIX} ${CMAKE_PREFIX_PATH})
 
+# Triton CC Client Libraries
+find_library(TRITON_HTTP_STATIC_LIB
+  NAMES httpclient_static
+  PATHS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+  REQUIRED
+)
+find_library(TRITON_GRPC_STATIC_LIB
+  NAMES grpcclient_static
+  PATHS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+  REQUIRED
+)
+find_library(TRITON_JSON_STATIC_LIB
+  NAMES json_utils_static
+  PATHS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+  REQUIRED
+)
+find_library(TRITON_SHM_STATIC_LIB
+  NAMES shm_utils_static
+  PATHS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+  REQUIRED
```

Comment:
I actually have a separate branch that I'm trying to verify that doesn't contain these changes. They don't seem necessary/in-scope with the goal of this PR.

Reply:
Appreciate your effort.
I recall I have tried to handle it here: 
https://github.com/triton-inference-server/perf_analyzer/pull/62

Reply:
The pipeline without these changes failed and reminded me why they're in place. It is again related to RHEL placing things in a `lib64` folder.

Without these changes, CMake was able to find these libs if they were in the ${CMAKE_INSTALL_PREFIX}/lib, but will fail to find it if it's in ${CMAKE_INSTALL_PREFIX}/lib64. This is a platform-agnostic way of checking both folders.

Reply:
This comment not about inclusion `lib64`, but about voiding compilation of entire `client` repository in favor of specific targets.

Regarding `lib64` inclusion CMake has it's own mechanism: 
https://cmake.org/cmake/help/latest/prop_gbl/FIND_LIBRARY_USE_LIB64_PATHS.html

Reply:
Thanks for showing me that mechanism. I tried adding it to my other branch but it still failed. I'm assuming it's because this property is only utilized when `find_library` is invoked (which I don't do in the other branch because then it would be the exact same as this branch). I believe `PATHS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"` would accomplish the same thing here, but would you like me to use the `FIND_LIBRARY_USE_LIB64_PATHS` method instead?

In terms of your request for a follow-up ticket can you clarify:
> expose targets...voiding compilation of entire client repository in favor of specific targets.

I believe we each tried a method of avoiding a client rebuild when building PA. We decided against my approach of a user-provided path to the necessary deps because they may be stale and you closed your ticket because I believe it was too much overhead for the PA team to handle at the time. We could create a ticket, however, I don't see a solution presenting itself until we change the Triton build architecture at-large first.

Reply:
@mc-nv please confirm how you would like me to proceed here--specifically my comments in the first paragraph--or if I can resolve the thread.